File: backend/app/Models/Article/ArticleCategory.php
<?php

namespace App\Models\Article;

use Eloquent;
use Illuminate\Database\Eloquent\Builder;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\HasMany;

/**
 * @mixin Eloquent
 * @mixin Builder
 */
class ArticleCategory extends Model
{
  protected $fillable = [
    'name',
    'code',
  ];

  public function articles(): HasMany
  {
    return $this->hasMany(Article::class);
  }
}

File: UserPanel/Models/ArticleCategory.cs
using System.Text.Json.Serialization;
namespace UserPanel.Models
{
    public class ArticleCategory
    {
        [JsonPropertyName("id")] public required int Id { get; set; }
        [JsonPropertyName("name")] public required string Name { get; set; }
        [JsonPropertyName("code")] public required string Code { get; set; }
    }
}
